src/views/IDUploader/index.jsx

```diff
@@ -74,23 +74,25 @@ class IDUploader extends Component {
     }
   };
 
-  async postCroppedImage(croppedImage, attemptNumber) {
-    let profile = await user.getProfileInfo();
-    let logMessage = `ID photo submission #${attemptNumber} for ${
-      profile.fullName
-    } from ${errorLog.parseNavigator(navigator)}`;
-    try {
-      await user.postIDImage(croppedImage);
-      this.setState({ submitDialogOpen: true });
-    } catch (error) {
-      logMessage += `, but image failed to post with error: ${error}`;
-      if (attemptNumber < 5) {
-        this.postCroppedImage(croppedImage, attemptNumber + 1);
-      } else {
-        this.setState({ errorDialogOpen: true });
+  async postCroppedImage(croppedImage) {
+    const profile = await user.getProfileInfo();
+    const userAgentData = logging.parseUserAgentData();
```

Comment:
This variable name does not fully describe what the function returns. I think this might be slightly clearer.
```suggestion
    const userAgentString = logging.parseUserAgentData();
```

Reply:
Hmm, to me the purpose of the variable name is to describe what the variable represents, not what kind of data it is - that's what we have typing and intellisense for. I'm not sure why `userAgentString` is clearer or more descriptive then `userAgentData`, and in fact the former seems somewhat misleading because it implies that we're talking about the `navigator.userAgent` string (as that's what most people mean when they say user agent string), which ideally we are not.